yammbs/_tests/unit_tests/test_minimize.py

```diff
@@ -134,6 +142,7 @@ def return_no_paths():
     )
 
 
+@pytest.mark.timeout(60 if platform.system() == "Darwin" else 30)
```

Comment:
See this comment for a justification of these timings, which should roughly be consistent with the timings in logs of this PR

https://github.com/openforcefield/yammbs/pull/36#issuecomment-2027624718